src/lib/ScannerRunCommand.ts

```diff
@@ -31,6 +31,12 @@ export abstract class ScannerRunCommand extends ScannerCommand {
 	 */
 	protected static flagsConfig = {
 		verbose: flags.builtin(),
+		// BEGIN: Filter-related flags.
```

Comment:
Move `--category` up to the common class so it's present in both command variants.